roles/hummingbird-common/templates/etc/hummingbird.conf.j2

```diff
@@ -2,12 +2,14 @@
 swift_hash_path_prefix = {{ hash_prefix }}
 swift_hash_path_suffix = {{ hash_suffix }}
 
-[storage-policy:0]
+{% for policy in storage_policies %}
+[storage-policy:{{loop.index0}}]
 default = yes
```

Comment:
I'm guessing they shouldn't all be default.

Reply:
oops, thanks for catching that.